src/monster.rs

```diff
@@ -124,10 +124,7 @@ impl Monster {
             Signpost => Behavior::Immobile,
         };
 
-        let invincible = match kind {
-            Npc | Signpost => true,
-            _ => false,
-        };
+        let invincible = matches!(kind, Npc | Signpost);
```

Comment:
I completely forgot `matches!` is a thing now! This is a very old codebase :-) (on Rust timeframes).